integration/testutil.go

```diff
@@ -607,6 +608,19 @@ func getManifestDigest(sh *shell.Shell, ref string, platform spec.Platform) (str
 	return "", fmt.Errorf("could not find manifest for %s for platform %s", ref, platforms.Format(platform))
 }
 
+func getReferrers(sh *dockershell.Shell, regConfig registryConfig, imgName, digest string) (*spec.Index, error) {
```

Comment:
Just noting that this only works for registries that support the OCI distribution spec v1.1.

Reply:
Yup. If we ever need to get referrers from a 1.0 registry within one of our tests we can replace the curl call with  `oras discover` cli command instead (we will have to add oras binary to our Dockerfile though)